testing/src/test/java/org/motechproject/nms/testing/it/kilkari/SubscriptionServiceBundleIT.java

```diff
@@ -760,4 +762,22 @@ public void testDeleteOldCompletedSubscription() {
         subscriber = subscriberDataService.findByCallingNumber(2000000000L);
         assertEquals(1, subscriber.getSubscriptions().size());
     }
+    
+    @Ignore
```

Comment:
Please add a JIRA link here to the bug created, assuming that is the reason why this is marked as Ignore